addOns/automation/src/main/java/org/zaproxy/addon/automation/ExtensionAutomation.java

```diff
@@ -208,23 +212,32 @@ private AutomationProgress runPlan(LinkedHashMap<?, ?> envData, ArrayList<?> job
                             Constant.messages.getString("automation.error.job.data", paramsObj));
                     continue;
                 }
-                job.applyParameters((LinkedHashMap<?, ?>) paramsObj, progress);
-
-                // In case errors or warnings are encountered in the applyParameters() call above
-                if (env.isTimeToQuit()) {
-                    break;
-                }
-
-                progress.info(
-                        Constant.messages.getString("automation.info.jobstart", job.getType()));
-                job.setEnv(env);
```

Comment:
I may be missing something, but I do not see this added in the loop below, was that on purpose? I was rebasing the tests PR on top of this one and discovered this...

Reply:
Yup, seems like I missed adding this, though it should be added right above `job.verifyParameters((LinkedHashMap<?, ?>) paramsObj, progress);`.That way both verification and the actual applying of parameters takes into account the resolved values.

Reply:
Fixed in #2979.